tests/testthat/test-print.R

```diff
@@ -5,7 +5,10 @@ test_that("print.draws_matrix runs without errors", {
   )
 
   x <- weight_draws(x, rep(1, ndraws(x)))
-  expect_output(print(x), "hidden reserved variables \\{'\\.log_weight'\\}")
+  expect_output(
+    print(x),
+    "hidden reserved variables ..\\.log_weight.."
```

Comment:
This is much more ugly to read in the tests, but should avoid problematic matching with `'` and `{`.

There are a lot of these.